apps/prairielearn/src/server.ts

```diff
@@ -2331,11 +2331,42 @@ if ((esMain(import.meta) || (isHMR && !isServerInitialized())) && config.startSe
         directories: [path.join(import.meta.dirname, 'migrations'), SCHEMA_MIGRATIONS_PATH],
         project: 'prairielearn',
       });
+    }
+    // We create and activate a random DB schema name
```

Comment:
We want to do the sproc init here so that the `sprocs` are initialized when we typecheck. In production, it doesn't matter to do this an extra time.